_pages/how-we-work/tools/text-editors.md

```diff
@@ -12,6 +12,7 @@ We use text editors to write documentation in [Markdown](https://github.com/adam
 * [Emacs](https://www.gnu.org/software/emacs/): Installed on OS X and Linux
   by default. Also available on OS X via [Homebrew](http://brew.sh/)
 * [Atom](https://atom.io/)
+* [Visual Studio Code](https://code.visualstudio.com/): VS Code is very extensible with lots plugins. One great plugin for pairing is [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare), which supports live editing of the same file, sharing a terminal session, port forwarding to the host's computer, and even audio.
```

Comment:
Given that Live Share is a plugin that creates a new connection between machines, we should submit it for approval. Game to do so?

https://handbook.18f.gov/software/#step-1-find-or-get-an-ato

In the meantime, ok with removing that part?

Reply:
Good catch! Updated, and I'll go figure out the approvals process. 😄